profile/src/main/java/org/apache/karaf/profile/assembly/ConfigInstaller.java

```diff
@@ -93,7 +99,54 @@ public void installConfigs(Feature feature, Downloader downloader, ArtifactInsta
                     }
                 }
             }
+            for (ConfigFile configFile : content.getConfigfile()) {
+                if (pidMatching(FilenameUtils.getBaseName(configFile.getFinalname()))) {
+                    installConfig(downloader, configFile);
+                }
+            }
+        }
+    }
+
+    private void installConfig(Downloader downloader, ConfigFile pConfigFile) throws Exception {
+        String path = pConfigFile.getFinalname();
+        if (path.startsWith("/")) {
+            path = path.substring(1);
+        }
+
+        Path configFileTarget = homeDirectory.resolve(substFinalName(path));
```

Comment:
I don't think it's a good idea to do this here. We should just copy the config file in the `etc` directory "raw".

Reply:
What do you mean by 'raw'? Do not use variables and the like? See comment / documentation above. It can be absolute or relative and use variables.